src/schemaToPropTypes.js

```diff
@@ -181,13 +230,18 @@ const propertiesReducer = (properties, requiredProps) => (str, propertyName) =>
  * @returns {string}
  */
 const getPropTypes = (schemaName, schema) => {
+	const isObjectDefinition = schema.type === 'object';
 	const requiredProps = 'required' in schema && schema.required;
-	const propTypeStringIndented = useIndentation(propTypeString);
-	const reducer = propertiesReducer(schema.properties, requiredProps);
+	const propTypeStringIndented = useIndentation(
+		propTypeString,
+		!!isObjectDefinition,
+		!isObjectDefinition ? false : undefined,
```

Comment:
Are you setting it to `undefined` to force default behaviour (that is: The opposite of whatever `prefix` is set to)?

If so, why don't you pass it directly, since you know the opposite (that is derived from the value that `isObjectDefinition` has)? Seems to be more straight forward to me.

Reply:
solved